contracts/core/Resolving/SportsAMMV2ResultManager.sol

```diff
@@ -456,6 +509,99 @@ contract SportsAMMV2ResultManager is Initializable, ProxyOwned, ProxyPausable, P
                 );
     }
 
+    function _exerciseLosingTickets(
+        address[] memory _tickets,
+        uint _numOfTicketsToExercise
+    ) internal returns (uint numOfTicketsToExercise) {
+        ISportsAMMV2 sportsAMM = ISportsAMMV2(manager.sportsAMM());
+        numOfTicketsToExercise = _numOfTicketsToExercise;
+        for (uint i; i < _tickets.length; i++) {
+            if (numOfTicketsToExercise == 0) {
+                break;
+            }
+            Ticket ticket = Ticket(_tickets[i]);
+            if (!ticket.isUserTheWinner() && ticket.isTicketExercisable()) {
```

Comment:
for consistency with LP contract, please switch these two conditions.
    ` if (ticket.isTicketExercisable()&&!ticket.isUserTheWinner())`